### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alex-hse-repository**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,11 +46,10 @@ from etna.models import ProphetModel
 
 # Read the data
 df = pd.read_csv("example_dataset.csv")
```
Comment (julia-shenshina) [inline README.md:48]: I think this path should be smth like `examples/data/example_dataset.csv`, shouldn't it?

Comment (alex-hse-repository) [inline README.md:48]:
```suggestion
df = pd.read_csv("examples/data/example_dataset.csv")
```

Comment (alex-hse-repository) [reply README.md:48]: Sounds reasonable)

**Review by julia-shenshina — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -45,12 +45,11 @@ from etna.datasets.tsdataset import TSDataset
...
 
 # Read the data
-df = pd.read_csv("example_dataset.csv")
-df["timestamp"] = pd.to_datetime(df["timestamp"])
+df = pd.read_csv("examples/data/example_dataset.csv")
 
 # Create a TSDataset
 df = TSDataset.to_dataset(df)
-ts = TSDataset(df,freq='1d')
+ts = TSDataset(df,freq="D")
```
Comment (julia-shenshina) [inline README.md:52]:
```suggestion
ts = TSDataset(df, freq="D")
```

**Review by julia-shenshina — APPROVED** [review]

**Pull request merged**